Makefile

```diff
@@ -150,9 +150,13 @@ create-cluster:
 deploy:
 	until svcat version | grep -m 1 'Server Version: v' ; do sleep 1 ; done
 	kubectl create namespace minibroker
-	helm upgrade --install minibroker --namespace minibroker \
-	--recreate-pods --force charts/minibroker \
-	--set image="$(IMAGE):$(TAG)",imagePullPolicy="$(IMAGE_PULL_POLICY)",deploymentStrategy="Recreate"
+	helm upgrade minibroker \
+		--install \
+		--force \
+		--recreate-pods \
+		--namespace minibroker \
+		--set image="$(IMAGE):$(TAG)",imagePullPolicy="$(IMAGE_PULL_POLICY)",deploymentStrategy="Recreate" \
```

Comment:
Can we just have multiple `--set`s?

Reply:
Fixed with https://github.com/kubernetes-sigs/minibroker/pull/56/commits/9fa275e4bc7295cedf96fceb96776ef9eb9205a9.